src/main/kotlin/apply/ui/admin/selections/SelectionView.kt

```diff
@@ -72,30 +73,31 @@ class SelectionView(
     }
 
     private fun createGrid(applicants: List<ApplicantResponse>): Component {
+        val applicationForms: HashMap<Long, ApplicationForm> = HashMap()
         return Grid<ApplicantResponse>(10).apply {
             addSortableColumn("이름", ApplicantResponse::name)
             addSortableColumn("이메일", ApplicantResponse::email)
             addSortableColumn("전화번호", ApplicantResponse::phoneNumber)
             addSortableColumn("성별") { it.gender.title }
             addSortableDateColumn("생년월일", ApplicantResponse::birthday)
+            addSortableDateTimeColumn("지원 일시") {
+                val applicationForm = applicationFormService.getByRecruitmentIdAndApplicantId(recruitmentId, it.id)
+                applicationForms[it.id] = applicationForm
+                applicationFormService.getByRecruitmentIdAndApplicantId(recruitmentId, it.id).submittedDateTime
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
                applicationForm.submittedDateTime
```